src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -286,7 +286,7 @@ public virtual string NormalizeKey(string key)
             ThrowIfDisposed();
             await Store.SetRoleNameAsync(role, name, CancellationToken);
             await UpdateNormalizedRoleNameAsync(role);
-            return await LogResultAsync(IdentityResult.Success, role);
+            return await Task.FromResult(IdentityResult.Success).WithLoggingAsync(this, role);
```

Comment:
Why isn't this using the version of `WithLoggingAsync()` that takes an `IdentityResult` as opposed to `Task<IdentityResult>`?